src/nvim/os/pty_process_win.c

```diff
@@ -12,6 +12,7 @@
 #include "nvim/memory.h"
 #include "nvim/mbyte.h"  // for utf8_to_utf16, utf16_to_utf8
 #include "nvim/os/pty_process_win.h"
+#include "nvim/os/os_win_conpty.h"
```

Comment:
Let's try to follow the naming pattern of other `os/*_win.{h,c}` files. Instead of `os/os_win_conpty.h`, I suggest:

    os/pty_conpty_win.h

That helps discovery because `pty_*` files will sort next to each other.